sys/auto_init/netif/auto_init_winc1500.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2017 Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup auto_init_gnrc_netif
+ * @{
+ *
+ * @file
+ * @brief   Auto initialization for WINC1500 Wi-Fi devices
+ *
+ * @author  Bumsik Kim <k.bumsik@gmail.com>
+ */
+
+#ifdef MODULE_WINC1500
+
+#include "log.h"
+#include "debug.h"
+#include "winc1500.h"
+#include "winc1500_params.h"
+#include "net/gnrc/netdev.h"
+#include "net/gnrc/netdev/eth.h"
+
+/**
+* @name   Define stack parameters for the MAC layer thread
+* @{
+*/
+#define WINC1500_MAC_STACKSIZE    (THREAD_STACKSIZE_DEFAULT + DEBUG_EXTRA_STACKSIZE)
+#ifndef WINC1500_MAC_PRIO
+#define WINC1500_MAC_PRIO         (GNRC_NETDEV_MAC_PRIO)
+#endif
+/** @} */
+
+extern winc1500_t winc1500; /** Located in winc1500.c */
```

Comment:
Here you should define an array of devices instead of accessing an external variable

Reply:
Please see my other comments. I need to discuss about it.